### ISSUE TITLE
paket update removes app.config redirects, paket install re-adds them

### ISSUE THREAD

**Issue by 0x53A** [fsprojects/Paket#2240]
I have forced redirects for a few annoying assemblies:

```
nuget FSharp.Core 4.1.0 redirects:force
nuget Newtonsoft.Json 9.0.1 redirects:force
```

This is the diff of my app.config after running update:
![image](https://cloud.githubusercontent.com/assets/4236651/24958109/81556e6a-1f8e-11e7-875e-8c2ee7110795.png)

This is the diff of the same file, after running install (without changing anything else):
![image](https://cloud.githubusercontent.com/assets/4236651/24958142/9a42dd18-1f8e-11e7-9b8d-c39e4d6aff93.png)

Combined, they are a no-op.

Note that this also happens on the first install after changing anything in paket.dependencies. The first install will then update the lockfile and remove all bindings. The second install will re-add the bindings.

### Known workarounds

run install again after update

### Version
> Paket version 4.4.0

**Comment by forki** [comment]
@gimpf is that what you wanted to fix? seems it's still there

**Comment by gimpf** [comment]
@forki No, I was seeing a different issue. Sorry I'm busy right now, but I should be able to provide a simplified example for the issue I was seeing once I've got some time. I do remember having some strange issue that I had to fix by deactivating the package directory garbage collection, but I'm unsure if it was related to this...
